modules/nw-networkpolicy-allow-internet.adoc

```diff
@@ -0,0 +1,107 @@
+
+.Procedure
+
+. Create the following `ingress-access` network policy to allow pods with the `networking/allow-ingress-access` label to receive traffic from outside sources. Save the YAML in the `ingress-access.yaml` file:
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: ingress-access
+spec:
+  podSelector:
+    matchLabels:
+      networking/allow-ingress-access: "true" <1>
+  policyTypes:
+  - Ingress
+  ingress:
+  - {}
+----
+<1> Apply this label to pods to enable the pods to receive traffic from outside sources.
+
+. Apply the network policy to the `project-a` namespace by entering the following command:
++
+[source,terminal]
+----
+$ oc apply -f ingress-access.yaml -n project-a
+----
+
+. Apply the network policy to the `project-b` namespace by entering the following command:
++
+[source,terminal]
+----
+$ oc apply -f ingress-access.yaml -n project-b
+----
+
+. Apply the `networking/allow-ingress-access=true` label to pods that must receive outside traffic by entering the following command:
++
+[source,terminal]
+----
+$ oc label pod busybox-pod-a networking/allow-ingress-access=true -n project-a
+----
++
+Repeat this step for all pods that must receive outside traffic.
+
+.Verification
+
+. Obtain the IP addresses of pods in `project-a` by running the following command:
++
+[source,terminal]
+----
+$ oc get pod -n project-a -o wide
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME            READY   STATUS    RESTARTS   AGE   IP            NODE                           NOMINATED NODE   READINESS GATES
+busybox-pod-a   1/1     Running   0          13m   10.132.0.38   ip-10-0-132-187.ec2.internal   <none>           <none>
+test-pod-a      1/1     Running   0          13m   10.132.0.40   ip-10-0-132-187.ec2.internal   <none>           <none>
+----
+
+. Ensure that pods with the `networking/allow-ingress-access=true` label can receive two ICMP packets by entering the following command. If you followed these instructions, the `busybox-pod-a` pod in `project-a` can receive traffic from another pod. For example:
++
+[source,terminal]
+----
+$ oc exec -it test-pod-b -n project-b -- ping -c 2 10.132.0.44
+----
++
+.Example output
++
+[source,terminal]
+----
+PING 10.132.0.44 (10.132.0.44): 56 data bytes
+64 bytes from 10.132.0.44: seq=0 ttl=42 time=1.137 ms
+64 bytes from 10.132.0.44: seq=1 ttl=42 time=0.672 ms
+----
+
+. Ensure that pods without the `networking/allow-ingress-access=true` label cannot receive traffic by entering the following command. If you followed these instructions, the `test-pod-a` pod in `project-a` cannot receive traffic. For example:
++
+[source,terminal]
+----
+$ oc exec -it busybox-pod-a -n project-a -- ping -c 2 10.132.0.40
+----
++
+.Example output
++
+[source,terminal]
+----
+PING 10.132.0.40 (10.132.0.40): 56 data bytes
+--- 10.132.0.40 ping statistics ---
+3 packets transmitted, 0 packets received, 100% packet loss
+----
```

Comment:
Add an example where ping passes as 
oc exec -it test-pod-a -n project-a -- ping -c 2 10.132.0.38
busybox-pod-a has the label networking/allow-ingress-access=true`

In this case if we are not sending traffic from pods in project-b to project-a, we could just get rid of project-b in example and just use pods in project-a to demonstrate traffic.